index.js

```diff
@@ -92,6 +92,9 @@ var tslintPlugin = function (pluginOptions) {
                 return cb(error, undefined);
             }
             var linter = getTslint(pluginOptions);
+            if (pluginOptions.configuration == null || typeof pluginOptions.configuration === "string") {
```

Comment:
Should use ``Object.prototype.toString.call(value)`` instead of the typeof operator.